Freebsd Question

I was getting closer to testing again on FreeBSD as I worked a little more on getting it compiled.

I found this old bug that I hit on the latest version:

https://github.com/sahib/rmlint/issues/139

```
gmake INTERNAL_FUSE=0
tools/update-version
c++ -O2 -g   -Wall -Wno-unused-result -I/usr/local/include/fuse -D_FILE_OFFSET_BITS=64  -DFUSE_USE_VERSION=29 -MMD -DUSE_XATTR=1 -DUGID_USE_RWLOCK=0  -c src/fs_cow.cpp -o obj/fs_cow.o
src/fs_cow.cpp:107:59: error: use of undeclared identifier 'O_LARGEFILE'
      src_fd = fs::open(src_fullpath_,O_RDONLY|O_NOFOLLOW|O_LARGEFILE);
                                                          ^
src/fs_cow.cpp:113:49: error: use of undeclared identifier 'O_LARGEFILE'
      dst_fd = fs::mktemp(dst_fullpath,O_WRONLY|O_LARGEFILE);
                                                ^
2 errors generated.
```

I am getting the same error. I tried the user's repo above from an older version and that compiled no problem:

```
./mergerfs --version
mergerfs version: 2.16.1-20-g12b9fc1
```

uname -a
FreeBSD freeplex 12.0-RELEASE FreeBSD 12.0-RELEASE r341666 GENERIC  amd64

Seems to be so close as the older version in my testing does some to do what I was hoping.

Any chance this can be addressed again?

--- a/src/fs_cow.cpp
+++ b/src/fs_cow.cpp
@@ -104,13 +104,13 @@ namespace fs
       int dst_fd;
       string dst_fullpath;
 
-      src_fd = fs::open(src_fullpath_,O_RDONLY|O_NOFOLLOW|O_LARGEFILE);
+      src_fd = fs::open(src_fullpath_,O_RDONLY|O_NOFOLLOW);
       if(src_fd == -1)
         return -1;
 
       dst_fullpath = src_fullpath_;
 
-      dst_fd = fs::mktemp(dst_fullpath,O_WRONLY|O_LARGEFILE);
+      dst_fd = fs::mktemp(dst_fullpath,O_WRONLY);
       if(dst_fd == -1)
         return cleanup_on_error(src_fd);
 

